examples/rect/README.md

```diff
@@ -11,11 +11,9 @@ theme: {
     '24': ['24px', '24px'],
   },
 }
-variants: {
-  rect: ['responsive'],
-},
 plugins: [
-  require('@viget/tailwindcss-plugins/rect')(),
+  require('./config/tailwind/rect'),
```

Comment:
nit this assumes but doesn't explain the pattern of putting tailwind plugins in separate files within ./config/tailwind

Reply:
Any suggestions? How about `require('./path/to/your/plugin/rect'),`? We just need to show that it's not part of the bundled plugins.